packages/linter/src/rules/MetadataIncludesOGImageSrc.ts

```diff
@@ -3,28 +3,28 @@ import { Rule } from "../rule";
 
 export class MetadataIncludesOGImageSrc extends Rule {
   run({ $ }: Context) {
-
     let hasOGImage = false;
 
-    $('meta').each(function (i, elem) {
-      if(elem.attribs.property === 'og:image' && elem.attribs.content) {
+    $("meta").each(function (i, elem) {
+      if (
+        elem.attribs.property &&
+        elem.attribs.property.includes("og:image", 0) &&
+        elem.attribs.content
+      ) {
         hasOGImage = true;
-        return false;             //break the loop
+        return false;
       }
     });
-    
 
-    return !(hasOGImage)
+    return !hasOGImage
       ? this.warn(`Missing og:image property or content source`)
       : this.pass();
   }
   meta() {
     return {
-      url:
-        "https://html.spec.whatwg.org/multipage/parsing.html#determining-the-character-encoding",
+      url: "https://ogp.me/",
```

Comment:
Would it besser to link to a stories specific documentation page?

Reply:
I can swap it for the [SEO for Stories post](https://blog.amp.dev/2020/02/12/seo-for-amp-stories/)